### DISCUSSION TITLE
Password authentication error despite using personal access token

### DISCUSSION THREAD

**Discussion by clemens-alexis-schroeder** [community/community#23925] [New to GitHub]
<p>After generating a personal access token and successfully using that token to authenticate, i still get this error as soon as i try to store the password with “git config credential.helper store”</p>
<p>Password authentication is temporarily disabled as part of a brownout. Please use a personal access token instead.</p>
<p>I even removed and newly pulled the repository, i don’t know what to do. I am on MacOS 10.15.7</p>

**Comment by Jef-Theysmeyer** [comment]
<p>I saw this as well in a couple of places - absolutely had a Personal Access token set</p>

**Comment by jsoref** [comment, marked as answer]
<p><a href="https://docs.github.com/en/get-started/getting-started-with-git/updating-credentials-from-the-macos-keychain" class="onebox" target="_blank" rel="noopener nofollow ugc">https://docs.github.com/en/get-started/getting-started-with-git/updating-credentials-from-the-macos-keychain</a></p>

**Comment by clemens-alexis-schroeder** [comment]
<p>Excuse me, but did you actually read my question?<br>
I AM already using personal access token.</p>
<p>“That’s It. Your error will be solved.”</p>
<p>Unfortunately, it’s not, and if it was the case, there’d be no need for this ticket in the first place…</p>

**Comment by jsoref** [comment]
<p>I’d love to know if this helps / is confusing / doesn’t work.</p>

**Comment by clemens-alexis-schroeder** [comment]
<p>Yes, that link actually helped, thank you! Removing the credentials from the keychain again (and also newly authenticating the Atom application) actually worked, fingers crossed…</p>

**Comment by jyotisourcecode** [comment]
<p>Github had stopped password authentication.<br>
You can check the solution in the given link</p><div class="onebox lazyYT lazyYT-container" data-youtube-id="aKaYpl-ZpGg" data-youtube-title="Solved - Support for password authentication was removed - Github - Mac and Windows" data-parameters="feature=oembed&amp;wmode=opaque">
  <a href="https://www.youtube.com/watch?v=aKaYpl-ZpGg" target="_blank" rel="noopener nofollow ugc">
    <img class="ytp-thumbnail-image" src="https://user-images.githubusercontent.com/11908379/181080050-38c7e90d-d9cd-4a00-983a-64e1ddd5208f.jpeg" title="Solved - Support for password authentication was removed - Github - Mac and Windows" width="480" height="360">
  </a>
</div>
